config/cscs.py

```diff
@@ -660,10 +660,10 @@
             ]
         },
         {
-            'name': 'pilatus',
-            'descr': 'Alps Cray EX Supercomputer (TDS)',
+            'name': 'eiger',
```

Comment:
I don't understand this. Pilatus becomes Eiger and Eiger Pilatus?

Reply:
I just moved Pilatus after Eiger, since we have also Dom after Piz Daint.